src/sagemaker/tensorflow/README.rst

```diff
@@ -860,7 +860,7 @@ The TensorFlow Docker images support Python 2.7. They include the following Pyth
 - tensorflow
 - tensorflow-serving-api
 
-The Docker images extend Ubuntu 16.04.
+The Docker images extend Ubuntu 16.04. GPU image uses CUDA 9.0.
```

Comment:
"The CPU Docker images extend the Ubuntu 16.04 Docker image. The GPU images extend the CUDA 9.0 Docker image."

Reply:
Both CPU and GPU extend ubuntu 16.04 image :)

Reply:
then just add CUDA to the list above and change "Python packages" to "dependencies". Or add a table like the other READMEs have